test/test_set_sequence_nr.py

```diff
@@ -0,0 +1,52 @@
+# Owner(s): ["module: autograd"]
```

Comment:
Please move this to `test/autograd/test_seq_nr.py` and import it at the bottom of `test/test_autograd.py`.
Feel free to just add it to `test/test_autograd.py` given that it is very small.